commons/commons-serialization/src/main/kotlin/ac/uk/ebi/biostd/exception/InvalidFileListException.kt

```diff
@@ -7,5 +7,8 @@ class InvalidFileListException(
     companion object {
         fun directoryCantBeFileList(fileName: String) =
             InvalidFileListException(fileName, "A directory can't be used as File List")
+
+        fun emptyFileList(fileName: String) =
+            InvalidFileListException(fileName, "A file list should contain at least one file")
```

Comment:
I think we should include the file list failing in case user has many they can identify the one failing.

Reply:
This class already does that, the full error message would be:

```
Problem processing file list 'empty.tsv': A file list should contain at least one file
```

Reply:
nice we amy need to include them file name in message test assertion.